sale_product_packaging_container_deposit/tests/test_sale_product_packaging_container_deposit.py

```diff
@@ -267,7 +267,7 @@ def test_sale_product_packaging_container_deposit_quantities_case7(self):
         box_line = self.sale_order.order_line.filtered(
             lambda ol: ol.product_id == self.box
         )
-        (box_line | pallet_line).invalidate_cache()
+        (box_line | pallet_line).invalidate_model()
```

Comment:
Shouldn't be `invalidate_recordset`?

Reply:
I would go for `invalidate_recordset`

> Invalidate the cache of all records of ``self``'s model, when the cached values no longer correspond to the database values.

But I am fine with `invalidate_model` as the result with be the same.

Reply:
Changed it to `invalidate_recordset` and force pushed.
Ready for merge now.